katello-installer/katello-installer-base.spec

```diff
@@ -60,7 +60,7 @@ foreman-installer --scenario capsule --migrations-only > /dev/null
 Summary:   Installer scenario for Katello development setup from git
 Group:	   Applications/System
 Requires:  %{name} = %{version}-%{release}
-
+Requires:  foreman-installer-katello
```

Comment:
Sorry for the delay getting to this, mind putting a blank line beneath this to separate the description block?